napi-inl.h

```diff
@@ -425,6 +425,38 @@ inline std::u16string String::Utf16Value() const {
 // Object class
 ////////////////////////////////////////////////////////////////////////////////
 
+namespace helper {
+
+template <typename Key>
+class PropertyLValue {
+public:
+  operator Value() const;
+  PropertyLValue& operator=(Value value);
```

Comment:
I think this should have overloads for other types that can be assigned to a value, like `Object::Set()`.